Classes/Core/Testbase.php

```diff
@@ -164,29 +166,121 @@ public function removeOldInstanceIfExists($instancePath): void
 
     /**
      * Link TYPO3 CMS core from "parent" instance.
+     *
      * For functional and acceptance tests.
      *
      * @param string $instancePath Absolute path to test instance
-     * @throws Exception
-     * @return void
+     * @throws \InvalidArgumentException
+     * @throws \RuntimeException
      */
-    public function setUpInstanceCoreLinks($instancePath): void
+    public function setUpInstanceCoreLinks(string $instancePath): void
     {
+        $filesystem = new Filesystem();
+
+        if (!$filesystem->isAbsolutePath($instancePath)) {
```

Comment:
is there a reason you introduce this check here now?

Reply:
Yes, if it's not absolute we can get in trouble with symlinks created in the wrong folders.